Backoffice/GenerateForm/Strategies/AbstractBlockStrategy.php

```diff
@@ -27,7 +27,9 @@ public function getTemplate()
      */
     public function getDefaultConfiguration()
     {
-        return array();
+        return array(
+            'maxAge' => 600
```

Comment:
@alavieille : Could you put the default maxAge as an application parameter instead of being hardcoded here ? It could be usefull to allow its tunning per project.